python/ray/tune/config_parser.py

```diff
@@ -103,6 +103,21 @@ def make_parser(parser_creator=None, **kwargs):
         action="store_true",
         help="Whether to checkpoint at the end of the experiment. "
         "Default is False.")
+    parser.add_argument(
+        "--keep-checkpoints-num",
+        default=None,
+        type=int,
+        help="Number of last checkpoints to keep. Others get "
+             "deleted. Default (None) keeps all checkpoints."
+    )
+    parser.add_argument(
+        "--checkpoint-score-attr",
+        default=None,
```

Comment:
Does it make sense to default to "training_iteration"?

Reply:
Agreed!